src/modules/Dashboard/Dashboard.tsx

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
+import { FC } from 'react';
```

Comment:
@abhishek-01k folder name still not fixed. It needs to be modules/dashboard/Dashboard.tsx